new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/CdcPipelineConfigurator.java

```diff
@@ -90,14 +85,12 @@ public void stop() {
   }
 
   private void start() {
-    Optional
-            .ofNullable(cdcPipelineJsonProperties)
-            .map(propertyReader::convertPropertiesToListOfMaps)
-            .orElseGet(() -> {
-              createStartSaveCdcDefaultPipeline(defaultCdcPipelineProperties);
-              return Collections.emptyList();
-            })
-            .forEach(this::createStartSaveCdcPipeline);
+
+    if (rawUnifiedCdcProperties.getPipeline() == null || rawUnifiedCdcProperties.getPipeline().isEmpty()) {
```

Comment:
Ditto

Why is this code duplicated?

Reply:
I will fix. Condition is duplicated because one entry for reader, second for pipeline